### ISSUE TITLE
Question: How to set AllowRoot

### ISSUE THREAD

**Issue by mgdsxhy** [monitoringartist/zabbix-agent-xxl#17]
Hi 
  I'm useing the zabbix-agent-xxl monitor docker container. 
  I monitor the interface net metrics with the key:   docker.xnet[/{#HCONTAINERID},all,TX-OK]. But can't get any values.  
  I want modify the config file  zabbix_agent.cnf in the zabbix-agent-xxl container, But appear follow error:
# docker ps -a

fbf1e9182507        monitoringartist/zabbix-agent-xxl-limited:latest   "/zocker-xxl"            4 seconds ago       Up 3 seconds        0.0.0.0:10050->10050/tcp   zabbix-agent-xxl
# docker exec -it fbf1e9182507 /bin/bash

rpc error: code = 2 desc = oci runtime error: exec failed: exec: "/bin/bash": stat /bin/bash: no such file or directory

How can I do set the AllowRoot=1

**Comment by jangaraj** [comment]
I recommend to read readme. Answers are already there - https://github.com/monitoringartist/zabbix-agent-xxl

tl;dr;
- env variable `ZA_AllowRoot=1` - it's enabled by default and you can't override it
- docker.xnet metrics are not supported

=> Install zabbix agent and load module https://github.com/monitoringartist/zabbix-docker-monitoring
